testfiles/ASHRAE901_RetailStripmall_STD2019_Denver.idf

```diff
@@ -852,7 +852,7 @@
     Until: 24:00,23.89,      !- Field 23
     For: Sunday Holidays AllOtherDays, !- Field 25
     Until:  8:00,29.44,      !- Field 26
-    Until:  8:00,26.67,      !- Field 28
+    Until:  8:00,26.67,      !- Field 28 Is this supposed to be 9:00?
```

Comment:
Given the occupancy schedule pattern for Type1, Type2 and Type3 where Type1 occupancy occurs at 8 AM this look correct.

    Schedule:Compact,
      Type1_OCC_SCH,           !- Name
      For: Sunday Holidays AllOtherDays, !- Field 50
      Until: 08:00,0.0,        !- Field 51
      Until: 10:00,0.05,       !- Field 53

    Schedule:Compact,
      Type2_OCC_SCH,           !- Name
      For: Sunday Holidays AllOtherDays, !- Field 52
      Until: 09:00,0.0,        !- Field 53
      Until: 10:00,0.05,       !- Field 55

    Schedule:Compact,
      Type3_OCC_SCH,           !- Name
      For: Sunday Holidays AllOtherDays, !- Field 62
      Until: 10:00,0.0,        !- Field 63
      Until: 11:00,0.11,       !- Field 65

Reply:
It does? Two consecutive "Until 8:00, X" statements is correct? At the very least one of them is redundant. Which one though?

Reply:
Two consecutive "Until 8:00, X" statements is NOT correct input syntax (but it doesn't hurt anything). The point is that the 2nd 8:00 should not be 9:00 since occupancy starts at 8:00. I think the developer noticed that occupancy started at 8:00 and simply changed the time on the 2nd line from 9 to 8. So the question is did the developer want 29.44 C or 26.67 C as the Tstat temp from midnight to 8:00? Either way the occupied Tstat setting of 23.89 C should begin at 8:00.

    For: Sunday Holidays AllOtherDays, !- Field 25
    Until:  8:00,29.44,      !- Field 26
    Until:  8:00,26.67,      !- Field 28
    Until: 23:00,23.89,      !- Field 30
    Until: 24:00,23.89;      !- Field 32

Reply:
Or maybe the first `Until 8:00` should really be `Until 7:00`?